fe1-web/parts/lao/organizer/eventCreation/events/CreateElection.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
```

Comment:
Looks **GREAT**! I will add comments on this file for an issue I encountered which is **not** a priority at the moment. For the purpose of this PR I approve the changes since my remark is more of an idea for improvement on **next** PRs. 

---------------------------------------------------------
The issue is the following: 
Supposing I add 20 ballot options and then I would like to remove ballot option number 3. I would have to remove all the options I created after option 3 before removing option 3. This happens because there is only one **remove** button. One idea (which I will explain commenting this file) would be to render a specific _per-ballot-option_ **remove** button when adding ballot options and removing the general remove button. 
Unfortunately (I tried it locally) the **WideButtonView** is pretty bulky on the screen to have 20 of them, it creates confusion. So another idea will be to do the same but creating a new (_small_) button component which stands on the right side of the **TextInput** of each ballot option.

Reply:
Thanks for the review.
Great idea with the small button to remove a specific ballot option. That would make it much easier to use when creating elections with many options. 
This is a good point to have for future PR, as you mentioned. :)